Original patch from gentoo: ../../dev/crosstool-NG/ct-ng.trunk/patches/strace/4.5.17/150-undef-CTL_PROC.patch
-= BEGIN original header =-
diff -dur strace-4.5.16.orig/system.c strace-4.5.16/system.c
-= END original header =-
diff -durN strace-4.5.17.orig/system.c strace-4.5.17/system.c
--- strace-4.5.17.orig/system.c	2007-01-17 00:22:36.000000000 +0100
+++ strace-4.5.17/system.c	2008-10-24 21:30:13.000000000 +0200
@@ -1612,7 +1612,9 @@
 	{ CTL_KERN, "CTL_KERN" },
 	{ CTL_VM, "CTL_VM" },
 	{ CTL_NET, "CTL_NET" },
+#ifdef CTL_PROC
 	{ CTL_PROC, "CTL_PROC" },
+#endif
 	{ CTL_FS, "CTL_FS" },
 	{ CTL_DEBUG, "CTL_DEBUG" },
 	{ CTL_DEV, "CTL_DEV" },
